docs/guides/index.mdx

```diff
@@ -55,6 +70,11 @@ Creating utility-scale quantum applications generally requires a variety of comp
 - Persist long-running workloads in the cloud, even if your laptop is turned off
 - Deploy reusable programs in the cloud
 
+If you'd like to start using Qiskit Serverless right, install it with pip:
```

Comment:
@abbycross I tihnk this is wrong. More generally, I'm not sure if the install snippets throughout the file follow the right style?